lighthouse-core/audits/accessibility/dlitem.js

```diff
@@ -11,6 +11,20 @@
  */
 
 const AxeAudit = require('./axe-audit');
+const i18n = require('../../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of an accesibility audit that evaluates if all definition list item elements have a definition list parent element. This descriptive title is shown to users when no user action is required. */
```

Comment:
Call out that `<dl>` == definition list.

Reply:
done